diff --git a/node_modules/reading-time/README.md b/node_modules/reading-time/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/reading-time/index.d.ts b/node_modules/reading-time/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/reading-time/index.js b/node_modules/reading-time/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/reading-time/lib/reading-time.js b/node_modules/reading-time/lib/reading-time.js
old mode 100644
new mode 100755
index 3ff116c..db76e8c
--- a/node_modules/reading-time/lib/reading-time.js
+++ b/node_modules/reading-time/lib/reading-time.js
@@ -32,14 +32,16 @@ function readingTime(text, options) {
 
   // calculate the number of words
   for (i = start; i <= end;) {
-    for (; i <= end && !wordBound(text[i]); i++) ;
+    for (; i <= end && !wordBound(text[i]); i++);
     words++
-    for (; i <= end && wordBound(text[i]); i++) ;
+    for (; i <= end && wordBound(text[i]); i++);
   }
 
   // reading time stats
   var minutes = words / options.wordsPerMinute
-  var time = minutes * 60 * 1000
+  // Math.round used to resolve floating point funkyness
+  //   http://docs.oracle.com/cd/E19957-01/806-3568/ncg_goldberg.html
+  var time = Math.round(minutes * 60 * 1000)
   var displayed = Math.ceil(minutes.toFixed(2))
 
   return {
diff --git a/node_modules/reading-time/lib/stream.js b/node_modules/reading-time/lib/stream.js
old mode 100644
new mode 100755
diff --git a/node_modules/reading-time/license b/node_modules/reading-time/license
old mode 100644
new mode 100755
